styles.css

```diff
@@ -288,6 +288,34 @@ main {
   border-bottom-color: #EFB7EC;
 }
 
+.timer-countdown {
```

Comment:
Thank you so much for this incredible styling! I've learned so much from your expertise!